### PR TITLE
Added support for rippleColor in android

### PR DESCRIPTION
**Pull request by sarthak-d11**
Part of this: https://github.com/okwasniewski/react-native-bottom-tabs/issues/11

**Summary**: This PR updates the ripple color of tab bar click for Android implementations.

**Testing**:

Verified Ripple color changes on Android.
Added FourTabsRippleColor for visual changes.

Adding Video for reference

https://github.com/user-attachments/assets/7cb8e2eb-b56c-4546-824a-315b11bd80ca

### CODE REVIEW

File: android/src/main/java/com/rcttabview/RCTTabViewViewManager.kt [hunk 1]
```diff
@@ -66,6 +68,13 @@ class RCTTabViewViewManager :
   fun setIcons(view: ReactBottomNavigationView, icons: ReadableArray?) {
     view.setIcons(icons)
   }
+  @ReactProp(name = "rippleColor")
+  fun setRippleColor(view: ReactBottomNavigationView, rippleColor: Int?){
+    if(rippleColor!=null) {
+      val color = ColorStateList.valueOf(rippleColor)
+      view.setRippleColor(color)
+    }
+  }
```
Comment (okwasniewski) [inline android/src/main/java/com/rcttabview/RCTTabViewViewManager.kt:77]: Something is off with formating here. Can you reformat the code with Android Studio?

File: android/src/main/java/com/rcttabview/RCTTabView.kt [hunk 2]
```diff
@@ -122,6 +124,9 @@ class ReactBottomNavigationView(context: Context) : BottomNavigationView(context
       LABEL_VISIBILITY_AUTO
     }
   }
+  fun setRippleColor(color: ColorStateList){
+    itemRippleColor = color
+  }
```
Comment (okwasniewski) [inline android/src/main/java/com/rcttabview/RCTTabView.kt:129]: Same, formatting is off

**Review by okwasniewski — COMMENTED** [review]
Thanks for working on this!

Can you check how the ripple color looks like when using Material Design 3? 

I've added guide on how to change the styles here: https://github.com/okwasniewski/react-native-bottom-tabs/blob/docs/content-update/docs/docs/docs/guides/android-native-styling.md

(It doesn't work for expo yet, but it should work in Example App)

File: docs/docs/docs/guides/usage-with-react-navigation.md [hunk 3]
```diff
@@ -51,6 +51,10 @@ Default options to use for the screens in the navigator.
 
 Whether to show labels in tabs. Defaults to true.
 
+#### `rippleColor`
+
+change ripple color on tab press. (Android Only)
```
Comment (okwasniewski) [inline docs/docs/docs/guides/usage-with-react-navigation.md:56]:
```suggestion
Changes ripple color on tab press. (Android Only)
```

**Comment by sarthak-d11** [comment]
> Thanks for working on this!
> 
> Can you check how the ripple color looks like when using Material Design 3?
> 
> I've added guide on how to change the styles here: https://github.com/okwasniewski/react-native-bottom-tabs/blob/docs/content-update/docs/docs/docs/guides/android-native-styling.md
> 
> (It doesn't work for expo yet, but it should work in Example App)

I have tested it using material design 3

this is how it looks

https://github.com/user-attachments/assets/c3d1f23b-673c-4245-85af-0741780ce93d

**Review by okwasniewski — APPROVED** [review]
Thank you!

**Pull request merged**